do_cmake.sh

```diff
@@ -13,7 +13,8 @@ if [ -e $BUILD_DIR ]; then
     exit 1
 fi
 
-PYBUILD="2"
+PYBUILD="3"
+ARGS="-GNinja"
```

Comment:
Hi @tchaikov , this line result in a cmake issue, do_cmake won't generating MakeFile (in ubuntu 18.04), could you please take a look?

Reply:
@CongMinYin could you check out https://lists.ceph.io/hyperkitty/list/dev@ceph.io/thread/PX5ISAIMD2BOV272VSV74GZFDJIWWZOE/ ? i don't think `do_cmake.sh` is responsible to create `Makefile`. it's just a convenience tool for configuring the cmake build system. if you prefer using `make` as the generator, might want to add an option for it. but i wonder why?

Reply:
> @CongMinYin could you check out https://lists.ceph.io/hyperkitty/list/dev@ceph.io/thread/PX5ISAIMD2BOV272VSV74GZFDJIWWZOE/ ? i don't think `do_cmake.sh` is responsible to create `Makefile`. it's just a convenience tool for configuring the cmake build system. if you prefer using `make` as the generator, might want to add an option for it. but i wonder why?

Sorry,  I didn't find that Doc has been updated. Another question related is: when I use `ARGS="-DCMAKE_BUILD_TYPE=RelWithDebInfo" ./do_cmake.sh`, it will report "Warning: is a debug version".  But if I use .`/do_cmake.sh -DCMAKE_BUILD_TYPE=RelWithDebInfo`. It's all right, do you know why?

like this:
****
WARNING: do_cmake.sh now creates debug builds by default. Performance
may be severely affected. Please use -DCMAKE_BUILD_TYPE=RelWithDebInfo
if a performance sensitive build is required.
****

Reply:
@CongMinYin please check the source code of `do_cmake.sh`, it's just a small dumb script. i don't really think `ARGS` is designed to be part of the public interface of this script.